docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -309,6 +318,20 @@ repository as well as each subsequent push to each pull request.
 To pass information from the upstream patch to the downstream patch use
 [downstream_expansions.set](Project-Commands.md#downstream_expansionsset)
 
+**Example**:  to allow testing Spruce tasks as part of patches for the Evergreen project,  
+a configuration like this could be added to **Evergreen's project page:**
+
+![patch_trigger_alias_example.png](../images/patch-trigger-alias-example.png)
```

Comment:
Why are we using 1 here as a date cutoff? Should we maybe explain why or maybe remove it?